modules/introduction/partials/new-features-80.adoc

```diff
@@ -139,6 +139,68 @@ to create all replicas and partitions in the same iteration,
 rather than doing it asynchronously in the background.
 This leads to better utilization of DCP streams.
 
+
+[#section-new-features-800-roles]
+=== Roles
+
+To support new features, Couchbase Server 8.0 has added, removed, or changed some roles:
+
+https://jira.issues.couchbase.com/browse/MB-64960[MB-64960]::
+The new Application Telemetry Writer (`application_telemetry_writer`) role supports the new application telemetry feature.
+Users with this role can call the telemetry endpoint to report telemetry to Couchbase Server.
+
+https://jira.issues.couchbase.com/browse/MB-65045[MB-65045]::
+The new Query List Index (`query_list_index`) lets the user list indexes without being able to alter them. 
+
+https://jira.issues.couchbase.com/browse/MB-63584[MB-63584]::
+The new Query Manage System Catalog (`query_manage_system_catalog`) role lets the user manage the system catalogs created by the new query workload report feature.
+
+
+https://jira.issues.couchbase.com/browse/MB-59606[MB-59606]::
+Couchbase Server 8.0 adds, removes, and changes some roles to split user management privileges from other security administration privileges.
+These changes are necessary to allow users to backup and restore users without requiring them to have the Full Admin role. 
+
++
+The following two roles have been removed:
+
++
+--
+* Local User Security Admin (`security_admin_local`)
+* External User Security Admin (`security_admin_external`)
+--
++
+Couchbase Server adds the following three roles to replace the removed roles:
+
++
+--
+* Security Admin (`security_admin`) has all of the non-user-related security privileges as the old Local User Security Admin and External User Security Admin roles had.
+* Local User Admin (`user_admin_local`) lets a user manage users and groups defined in the local authentication domain.  
+It does not have the other security privileges that the old Local User Security Admin role.
+* External User Admin (`user_admin_external`) lets a user manage users defined in the external authentication domain. 
+It does not have the other security privileges that the old External User Security Admin had.
+--
++
+In addition, the Backup Full Admin (`backup_admin`) role now lets a user backup and restore users. 
```

Comment:
A user with the backup_admin role cannot backup/restore users unless they also also have one or both of user_admin_local, user _admin_external.  The backup_admin role has the same capabilities as prior to morpheus. With morpheus the user doesn't need "Full admin" to backup/restore users. It now needs a more limited additional role(s).